tests/test_integration.py

```diff
@@ -62,7 +62,7 @@ def test_sf_graph(self):
 
     def test_illegal_edges(self):
         """Run demo.py and check that no illegal edges are reported. This ensures the lagrange parameter is set appropriately."""
-        args = demo.read_in_args(["--graph", "internet"])
+        args = demo.read_in_args(["--graph", "karate"])
```

Comment:
Is this temporary that we will run tests on the karate graph? Since it's confirming the Lagrange parameter is set appropriately for the *default* run, it should remain as the internet graph for the long term. The Lagrange parameter value will be different for a graph on 33 nodes versus 1000 nodes.

Reply:
Ah, I see. No, this part wasn't meant to be temporary. I've reverted this, though perhaps some future work could be allowing users to pass in a Lagrange parameter.